monkey/infection_monkey/network/__init__.py

```diff
@@ -14,10 +15,27 @@ def is_host_alive(self, host):
 class HostFinger(object):
     __metaclass__ = ABCMeta
 
+    _SCANNED_SERVICE = ''
+
+    def format_service_info(self, port=None, url=None):
```

Comment:
Since there's no meaning to url in this context, timestamp isn't important and name is easy to set, I would remove this function or re-implement it so it just gets the service dictionary and port, and sets the port and name in the service dictionary.